### PR TITLE
Assistant fixes

### PR DESCRIPTION
**Pull request by otaj**
Fix assistant bug accidentally introduced in #15207 

cc @tchaton @rohitgr7 @carmocca @akihironitta @borda

### CODE REVIEW

File: .azure/app-cloud-e2e.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ trigger:
     - "src/lightning_app/**"
     - "examples/app_*"
     - "tests/tests_app_examples/**"
+    - ".actions/assistant.py"
```
Comment (justusschock) [inline .azure/app-cloud-e2e.yml:22]: shouldn't we rather retrigger everything when anything in `.actions` changes?

**Review by justusschock — APPROVED** [review]

File: .azure/app-cloud-e2e.yml (hunk 1 above)
Comment (otaj) [reply .azure/app-cloud-e2e.yml:22]: Probably yes

File: .github/checkgroup.yml [hunk 2]
```diff
@@ -10,6 +10,7 @@ subprojects:
     paths:
       - ".circleci/**"
       - ".github/workflows/ci-circleci.yml"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:13]: Not necessary since the `pytorch_lightning` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 3]
```diff
@@ -63,6 +65,7 @@ subprojects:
   - id: "pytorch_lightning: CPU"
     paths:
       - ".github/workflows/ci-pytorch-test-full.yml"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:68]: Not necessary since the `pytorch_lightning` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 4]
```diff
@@ -85,6 +88,7 @@ subprojects:
   - id: "pytorch_lightning: Slow"
     paths:
       - ".github/workflows/ci-pytorch-test-slow.yml"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:91]: Not necessary since the `pytorch_lightning` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 5]
```diff
@@ -94,18 +98,21 @@ subprojects:
     paths:
       - ".azure/gpu-tests-pytorch.yml"
       - "tests/tests_pytorch/run_standalone_*.sh"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:101]: Not necessary since the `pytorch_lightning` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 6]
```diff
@@ -94,18 +98,21 @@ subprojects:
...
     checks:
       - "pytorch-lightning (GPUs)"
 
   - id: "pytorch_lightning: Azure HPU"
     paths:
       - ".azure/hpu-tests.yml"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:108]: Not necessary since the `pytorch_lightning` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 7]
```diff
@@ -94,18 +98,21 @@ subprojects:
...
     checks:
       - "pytorch-lightning (HPUs)"
 
   - id: "pytorch_lightning: Azure IPU"
     paths:
       - ".azure/ipu-tests.yml"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:115]: Not necessary since the `pytorch_lightning` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 8]
```diff
@@ -115,6 +122,7 @@ subprojects:
       - ".github/workflows/docs-*.yml"
       - "requirements/docs.txt"
       - "requirements/pytorch/**"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:125]: Not necessary since the `pytorch_lightning` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 9]
```diff
@@ -194,6 +204,7 @@ subprojects:
     paths:
       - ".azure/gpu-tests-lite.yml"
       - "tests/tests_lite/run_standalone_*.sh"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:207]: Not necessary since the `lightning_lite` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 10]
```diff
@@ -227,6 +239,7 @@ subprojects:
   - id: "lightning_app: Azure"
     paths:
       - ".azure/app-cloud-e2e.yml"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:242]: Not necessary since the `lightning_app` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 11]
```diff
@@ -236,6 +249,7 @@ subprojects:
       - ".github/workflows/docs-*.yml"
       - "requirements/docs.txt"
       - "requirements/app/**"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:252]: Not necessary since the `pytorch_lightning` group covers it

```suggestion
```

File: .github/checkgroup.yml [hunk 12]
```diff
@@ -246,6 +260,7 @@ subprojects:
     paths:
       - ".github/workflows/code-checks.yml"
       - "pyproject.toml"  # includes mypy config
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:263]: are actions relevant for mypy?

File: .github/checkgroup.yml [hunk 13]
```diff
@@ -257,6 +272,7 @@ subprojects:
       - "src/lightning/**"
       # all __about__, __version__, __setup__
       - "src/*/__*.py"
+      - ".actions/**"
```
Comment (carmocca) [inline .github/checkgroup.yml:275]: You can remove actions/setup_tools.py a few lines above

**Review by carmocca — APPROVED** [review]

**Comment by carmocca** [comment]
This PR will need all of CI to be green to get merged. Hopefully the flake gods are happy with us

**Review by awaelchli — APPROVED** [review]

**Pull request merged**